### ISSUE TITLE
https instead of http

### ISSUE THREAD

**Issue by markus2120** [exebetche/vlsub#187]
Could you move all requests to https?
Thanks

**Comment by mtrz** [comment]
No, that won't work so easy. You need a real SSL parser in lua, https links uses also a different port 443, not 80 like http. So that would probably not work. I also playing around to get some SSL support in VLSub but without success for now. I want to fix the translation link that uses https.
Here are some more info about this SSL issue:
https://github.com/exebetche/vlsub/issues/197
